fontc/src/bin/fontc_bulk/README.md

```diff
@@ -0,0 +1,26 @@
+# fontc_bulk
+
+This is a binary for executing font compilation (and possibly other tasks) in
+bulk.
+
+It takes as input a JSON dictionary (as a text file) containing key/value pairs
+where the keys are the names of fonts and the values are the /username/repo
+portion of a github URL.
+
+This data is derived from the cache files used by `gfautobuilder.py`: see [this
+comment](https://github.com/googlefonts/fontc/issues/724#issuecomment-1977431334)
+for details on how to get or rebuild this. (We restructure the data slightly to
+include only the 'real_upstream' value.)
+
+We include a file containing information for all of the known google fonts font
+repositories as of June 2024, at resources/testdata/gf_fonts.json.
```

Comment:
Let's add `resources/scripts/make-me-that-file` instead of committing the giant json file.